testing/btest/Baseline/cluster.generic.errors/broker.out

```diff
@@ -0,0 +1,9 @@
+### BTest baseline data generated by btest-diff. Do not edit. Use "btest -U/-u" to update. Requires BTest >= 0.63.
```

Comment:
I haven't peeked ahead to Part 3 and 4 — will the test use `broker.*` and `zeromq.*` in the future? Just checking in case we're accidentally adding baselines from an earlier version of that test.

Reply:
No, this is a left over. Nice catch!